app/controllers/manage_cmr/permissions_controller.rb

```diff
@@ -148,12 +146,28 @@ def destroy
 
   private
 
+  def collection_permission_params
```

Comment:
Do we understand how the permit system works? If it is rerun, I assume it is recomputing.  In which case, I think an ivar would save compute in construct_request_object, for instance.